### DISCUSSION TITLE
share media

### DISCUSSION THREAD

**Discussion by pokhiii** [spatie/laravel-medialibrary#2628] [Q&A]
I have a warehouse project in which an item is created for each quantity number. So, if I say

- item = 'chairs`
- quantity = 5
- images = 2

this will create 5 new rows in the items table. each item will share the 2 image files. if i do not share then 2 images will be stored for each of the 5 items = 10 images!

is there a way to share single media with different model instances?

**Comment by PaolaRuby** [comment]
With Laravel's replicate model function
